spec/features/partner_employment_page_spec.rb

```diff
@@ -30,21 +30,21 @@
 
   context "when I have indicated that the partner is employed" do
     before do
-      fill_in_applicant_screen_without_passporting_benefits
-      select_applicant_boolean(:partner_employed, true)
-      select_applicant_boolean(:partner_over_60, false)
       click_on "Save and continue"
       travel_from_dependants_to_past_client_assets
+      select_boolean_value("partner-details-form", :partner_over_60, false)
+      select_boolean_value("partner-details-form", :partner_employed, true)
+      click_on "Save and continue"
     end
 
     it "shows the partner employment page" do
       expect(page).to have_content(partner_employment_page_header)
     end
 
-    it "has a back link to the assets page" do
-      click_link "Back"
-      expect(page).to have_content assets_page_header
-    end
+    # it "has a back link to the assets page" do
```

Comment:
I'm basically allergic to commented-out code. If the test works, it shouldn't be commented out. If it doesn't but should, let's fix it. If it doesn't work and shouldn't work, let's remove it.

Reply:
I meant to go back and fix this and couldn't find it. Updated the test and reinstated it